apps/evm/lib/evm/operation/system.ex

```diff
@@ -6,92 +6,22 @@ defmodule EVM.Operation.System do
   alias EVM.Memory
   alias EVM.ExecEnv
   alias EVM.Interface.AccountInterface
-  alias EVM.Interface.BlockInterface
-  alias EVM.Helpers
+  #  alias EVM.Interface.BlockInterface
+  #  alias EVM.Helpers
   alias EVM.Address
   alias EVM.Stack
   alias EVM.Operation
   alias Blockchain
 
-  @dialyzer {:no_return, callcode: 2}
```

Comment:
@hayesgm I've deleted these two functions because I think they're implemented elsewhere. I might be wrong (but they're also not used anywhere).
Is create/2 implemented as AccountInterface.create_contract/3?
Is callcode/2 implemented as AccountInterface.message_call/6?